Makefile

```diff
@@ -87,6 +87,10 @@ build-plugins: $(PLUGINS)
 build-check:
 	$(Q)$(GO_BUILD) -v $(GO_MODULES)
 
+mod-tidy:
+	$(Q)$(GO_CMD) mod tidy
```

Comment:
Is this step needed, the `go-mod-tidy` script handles the root go.mod file also.

Reply:
I'm not sure if the return order of `find . -name go.mod | while read -r gomod` is always deterministic and always return `./go.mod` first. If so we can just remove this.

 I called `go mod tidy` explicitly here to make sure the root module is `tidy`ed first before other modules that depend on it.